dune

```diff
@@ -1,12 +1,12 @@
 ; Default flags for all Coq libraries.
 (env
- (dev     (flags :standard -rectypes -w -9-27+40+60))
+ (dev     (flags :standard -rectypes -w -9-27+40+60 \ -short-paths))
```

Comment:
IMHO it would be good if we could comment a bit more the reasoning for the flags [in the style of what we do in configure]. It is important to document this stuff well.

In particular, we may add something signalling that `short-paths` is needed due to aliases, but would we remove the remaining aliases we could also add the flag back.